README.md

```diff
@@ -14,3 +14,51 @@ A very fast [golang](http://golang.org/) port of [thumbor](https://github.com/th
 - [ ] Other storage engines
 - [ ] Tests
 - [x] Unsafe mode
+
+## HOW-TO
+### DEPLOY
+Gothumb is on k8bs. Refer to [opendoor-gothumb](https://github.com/opendoor-labs/opendoor-gothumb)
+
+### TEST
+Testing in staging:
+
+Build the docker image by
+```
+docker build . -t opendoor/gothumb:staging
+```
+Log in to docker hub with the opendoor account, if you haven't done so by
+`docker login`. Creds are in 1Password.
+
+Push to docker hub
+`docker push opendoor/gothumb:staging`
+Deploy to staging by first getting the pod
+```
+kubectl get pods -n staging --selector="app=gothumb"
+```
+```
+NAME                       READY     STATUS    RESTARTS   AGE
+gothumb-3265306518-htm14   1/1       Running   0          4h
+```
+```
+kubectl delete pod gothumb-3265306518-htm14 -n staging
```

Comment:
is this missing the deploy command?

Reply:
deleting the pod should start a new pod, which should pull down the new image